### DISCUSSION TITLE
(cdburnerxp) suggest moving the package to manual

### DISCUSSION THREAD

**Discussion by TheCakeIsNaOH** [chocolatey-community/chocolatey-packages#2402] [General]
The cdburnerxp update script is currently failing due to a cerificate error. 
As per https://cdburnerxp.se/impressum the company behind it has shut down, so it is unlikely to be updated in the future. 
Thus, I think moving the package to be manual would make sense.

**Comment by AdmiringWorm** [comment]
I agree, as it is unlikely that there will be any updates (unless the company resurrects), it makes sense to move this package to be a manual package.

**Reply by AdmiringWorm** [reply]
I have created the issue #2427 based on this discussion post. If anyone is opening a PR, make sure the issue is referenced.
